src/main/java/com/rapid7/net/AsyncLogger.java

```diff
@@ -12,7 +12,7 @@
 import static com.google.common.base.Strings.isNullOrEmpty;
 
 /**
- * Logentries Asynchronous Logger for integration with Java logging frameworks.
+ * InsightOps Asynchronous Logger for integration with Java logging frameworks.
```

Comment:
What do you think of making this class immutable, and adding a builder?

Reply:
It has sense to have it immutable but I think is not possible for how is handled the configuration in LogentriesAppender for logback and log4j. These classes need setter to configure AsyncLogger with params from properties (or xml) file. I'll investigate if it can be done using constructor

Reply:
Using [https://dzone.com/articles/design-patterns-the-builder-pattern](Builder pattern)? Or does the configuration need to be changed after the object is created?

Reply:
I know what you mean, I was looking for a place where to instantiate the AsyncLogger object (using the builder) in classes like com.rapid7.log4j.LogentriesAppender and com.rapid7.logback.LogentriesAppender. I guess I can override the AppenderBase.start and AppenderSkeleton.activateOptions methods.